openapi/index.yaml

```diff
@@ -130,6 +130,46 @@ paths:
             $ref: '#/definitions/ProblemJson'
         '429':
           description: Too many requests
+    put:
```

Comment:
maybe it's more clear to add a new endpoint for message-status resource (in a REST manner) :)

Reply:
Ok, since we were changing something related to a message I thought it could be ok to "put" something for messages, but I was also thinking to create a new message-status endpoint, so I'll create it.